GUI/ManualAddressDialogUtils/PointerChainOffset.py

```diff
@@ -0,0 +1,62 @@
+from PyQt6.QtWidgets import QWidget, QFrame, QLabel, QLineEdit, QPushButton, QHBoxLayout, QSizePolicy, QSpacerItem
+from PyQt6.QtCore import Qt, pyqtSignal
+from operator import add as opAdd, sub as opSub
+
+from GUI.Utils import guiutils
+    
+
+# Only intended to be used by ManualAddressForm
+class PointerChainOffset(QFrame):
+    offset_changed_signal = pyqtSignal(name="offsetChanged")
+
+    def __init__(self, offset_index:int, parent:QWidget|None=None):
+        super().__init__(parent)
+        self.offset_index = offset_index
+        self.initUI()
+
+    def initUI(self):
+        offsetLayout = QHBoxLayout(self)
+        offsetLayout.setContentsMargins(0, 3, 0, 3)
+        self.setLayout(offsetLayout)
+        buttonLeft = QPushButton("<", self)
+        buttonLeft.setFixedWidth(20)
+        offsetLayout.addWidget(buttonLeft)
+        self.offsetText = QLineEdit(self)
+        self.offsetText.setValidator(guiutils.validator_map["int_hex"])
```

Comment:
`int_hex` doesn't include the character `-` but we need it for negative offsets. Two solutions come to mind:
- Create a new regex for this purpose
- Change `int_hex` to include `-`, it's currently only used in scan lineEdit when hex checkbox is checked. Do we need `-` also for hex entries in scan lineEdit? That's up to debate

Reply:
Going to add the - to the regex. Also adjusted search validation for this new behaviour.